crates/forge/tests/e2e/common/runner.rs

```diff
@@ -205,3 +207,85 @@ pub(crate) fn get_current_branch() -> String {
         String::from_utf8(output.stdout).unwrap().trim().to_string()
     }
 }
+
+pub(crate) fn intersection_with_wildcard(
+    asserted: &Vec<String>,
+    actual: &Vec<String>,
+) -> Vec<(String, usize)> {
+    let mut result = vec![];
+
+    for element in asserted {
+        let escaped = regex::escape(element);
+        let replaced = escaped.replace("\\[\\.\\.\\]", ".*");
```

Comment:
Doesn't it change the behavior of [..]? [..] is limited to one line, `.*` is not

Reply:
We are matching the output line by line, so it shouldn't be a problem.